src/component/content.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+const firstName = ['Yoda','Willy','fool','dilly','billy','Sam', ''];
+
+const lastName =['Sanchez','silly','meow','son','Roger'];
+
+const renderFirst= firstName[Math.floor(Math.random()* firstName.length)];
+    
+const rendeLast= lastName[Math.floor(Math.random()* lastName.length)];
+   
+const renderboth = renderFirst + " " + rendeLast;
+
+
+
+
+
+class Content extends Component{
+    constructor(props){
+        super(props);
+        this.state={};
+
+        this.handlename=this.handlename.bind(this);
+    }
+
+
+
+    handlename(props){
+        alert(renderboth)
+    }
+
+
+    render(props){
+        return(
+            <div className="container">
+                <div className="row">
+                    <div className="col-md-12 offset-4">
+                        <h1>Hello {}</h1>
+                    </div>
+                    <div className="col offset-4">
+                        <button className="btn btn-info" onClick={this.handlename}>Genarate A Name</button>
+                        <p>The button above Genarate a name for you! </p>        
```

Comment:
why dont we also display the genetared random name below for the user to see instead of having it be an alert pop up
```suggestion
                        <p>The button above Genarate a name for you! </p>  
                        <p>{this.state.randomName}</p>
                              
```

Reply:
Yes that was my initial idea but I didn't know how to implement it. I'll try this